airbyte-integrations/connectors/source-amplitude/source_amplitude/streams.py

```diff
@@ -68,10 +67,6 @@ def url_base(self) -> str:
         subdomain = "analytics.eu." if self.data_region == "EU Residency Server" else ""
         return f"https://{subdomain}amplitude.com/api/"
 
-    @property
-    def availability_strategy(self) -> Optional["AvailabilityStrategy"]:
-        return None
```

Comment:
Just double checked - I think @midavadim is right. The availability strategy's `get_first_record_for_slice` calls `stream.read_records` so the 404 error handling that returns empty records should work, and the AS should say "connected to stream, but got 0 records". So this should work in that case 👍🏻